karma.config.js

```diff
@@ -1,8 +1,27 @@
+const glob = require('glob');
+
 process.env.CHROME_BIN = require('puppeteer').executablePath();
 
+let testFilesGlob = 'test/**/*.spec.js';
+const globIndex = process.argv.indexOf('--glob');
+if (globIndex > -1) {
+  testFilesGlob = process.argv[globIndex + 1] || '';
+  if (!testFilesGlob.includes('**/')) {
+    testFilesGlob = `test/**/${testFilesGlob}`;
+  }
+  if (!testFilesGlob.endsWith('*.spec.js')) {
+    testFilesGlob = `${testFilesGlob}*.spec.js`;
+  }
+  if (glob.sync(testFilesGlob).length === 0) {
```

Comment:
Karma already does this. I think it's better to let Karma handle reporting/checks and us worry only about accepting a file pattern from the command line. Otherwise this can get out of sync as Karma evolves so maybe better to not do this check at all. The PR will then introduce only changes within the scope defined by the title of the PR.

Reply:
Mh... Calling Karma with a glob param with no matches looks like this:
```
$ yarn karma --glob foo
yarn run v1.22.4
$ karma start karma.config.js --single-run --browsers ChromeHeadless,FirefoxHeadless --glob foo
Testing files matching "test/**/foo*.spec.js"
06 07 2020 17:45:46.587:WARN [filelist]: Pattern "/Users/manuel.jasso/dev/github/secure-javascript-environment/test/**/foo*.spec.js" does not match any file.
06 07 2020 17:45:46.591:INFO [karma-server]: Karma v4.4.1 server started at http://0.0.0.0:9876/
06 07 2020 17:45:46.591:INFO [launcher]: Launching browsers ChromeHeadless, FirefoxHeadless with concurrency unlimited
06 07 2020 17:45:46.596:INFO [launcher]: Starting browser ChromeHeadless
06 07 2020 17:45:46.602:INFO [launcher]: Starting browser FirefoxHeadless
06 07 2020 17:45:47.003:INFO [HeadlessChrome 83.0.4103 (Mac OS X 10.15.5)]: Connected on socket RBQ-TuF0rPfL0cc7AAAA with id 87945352
HeadlessChrome 83.0.4103 (Mac OS X 10.15.5): Executed 0 of 0 SUCCESS (0.001 secs / 0 secs)
HeadlessChrome 83.0.4103 (Mac OS X 10.15.5): Executed 0 of 0 SUCCESS (0.001 secs / 0 secs)
Firefox 78.0.0 (Mac OS X 10.15.0): Executed 0 of 0 SUCCESS (0.001 secs / 0 secs)
TOTAL: 0 SUCCESS
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

But if we check first it looks like this:
```
 yarn karma --glob foo
yarn run v1.22.4
$ karma start karma.config.js --single-run --browsers ChromeHeadless,FirefoxHeadless --glob foo
No files matching "test/**/foo*.spec.js"
✨  Done in 0.51s.
```

I like the 2nd form better.